R/miniconda.R

```diff
@@ -0,0 +1,197 @@
+
+#' Install Miniconda
+#'
+#' Download the [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
+#' installer, and use it to install Miniconda.
+#'
+#' @param path The path in which Miniconda will be installed. Note that the
+#'   installer does not support paths containing spaces.
+#' 
+#' @param version The version of Python associated with the Miniconda installer
+#'   to be used.
+#' 
+#' @param update Boolean; update to the latest version of Miniconda after install?
+#' 
+#' @param python The version of Python to use. You can request specific
```

Comment:
These docs might be a little confusing - this is for switching to a different specific version of Python in the conda environment, right?